geopandas/tests/test_cmapplotting.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import, division
+
+import itertools
+import warnings
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib as mpl
+
+from shapely.geometry import Point, MultiPoint
+
+from geopandas import GeoSeries, GeoDataFrame, plotting, read_file
+from geopandas.datasets import get_path
+
+import pytest
+
+@pytest.fixture(autouse=True)
+def close_figures(request):
+    yield
+    plt.close('all')
+
+class TestCMapPlotting:
+
+    def setup_method(self):
+        self.N = 10
+        self.points = GeoSeries(Point(i, i) for i in range(self.N))
+
+        values = np.arange(self.N)*10
+        self.df = GeoDataFrame({'geometry': self.points, 'values': values})
+
+        bounds = np.array([-10, 0, 10, 30, 50, 70, 90])
+        self.norm = mpl.colors.BoundaryNorm(boundaries=bounds, ncolors=256)
+
+    def test_min_max(self):
+        ax = self.df.plot(column='values', norm = self.norm)
+
+        expected_lim = (self.df['values'].min(), self.df['values'].max())
+        actual_lim = ax.collections[0].get_clim()
+
+        assert expected_lim == actual_lim
```

Comment:
Here, I'm checking whether the actual limits obtained from the axes and the expected limits from the self.df are equal or not.